README.md

```diff
@@ -5,17 +5,51 @@ First, setup git and create a github account as explained in the slides.<br/>
 
 Execute the following steps to practice various topics covered in the intro to git session<br/>
 1) Fork the repository using the fork icon.<br/>
+   ![](images/fork.png)
 2) Clone the forked repository into a directory of your choice in your local machine using the command git clone <url_of_forked_repository> <br/>
+   ![](images/clone.png)
 3) Create a branch with name myFirstBranch using the command git branch <br/>
+   ![](images/git_branch.png)
```

Comment:
Seems like there is a lot of blank space above the image for 3 and 4. And 7 as well